internal/backends/linuxkms/display/swdisplay/linuxfb.rs

```diff
@@ -202,16 +331,28 @@ impl super::SoftwareBufferDisplay for LinuxFBDisplay {
         self.first_frame.set(false);
 
         match self.format {
-            drm::buffer::DrmFourcc::Xrgb8888 => {
-                // 32-bit format - no conversion needed
+            drm::buffer::DrmFourcc::Xrgb8888
+            | drm::buffer::DrmFourcc::Argb8888
+            | drm::buffer::DrmFourcc::Xbgr8888
+            | drm::buffer::DrmFourcc::Abgr8888
+            | drm::buffer::DrmFourcc::Rgba8888
+            | drm::buffer::DrmFourcc::Bgra8888 => {
+                // 32-bit formats
+                callback(self.back_buffer.borrow_mut().as_mut(), age, self.format)?;
```

Comment:
These calls look correct, but they'll fail at run-time as only Xrgb8888 and Rgb565 are supported by the Skia and Slint software renderers right now.